src/vs/platform/window/common/window.ts

```diff
@@ -152,6 +152,7 @@ export interface IWindowSettings {
 	readonly restoreFullscreen: boolean;
 	readonly zoomLevel: number;
 	readonly titleBarStyle: TitlebarStyle;
+	readonly titleControlsStyle: 'native' | 'custom' | 'hidden';
```

Comment:
Can we introduce an enum for this, similar to `TitlebarStyle`. Especially because `titleBarStyle` also has `native` and `custom`.

Reply:
Done